content/webapp/utils/iiif/v3/index.ts

```diff
@@ -547,8 +550,8 @@ export function getOriginalFiles(
     canvas.original.length > 0
       ? canvas.original
       : canvas.painting.length > 0
-        ? canvas.painting
-        : canvas.supplementing;
+      ? canvas.painting
+      : canvas.supplementing;
```

Comment:
Very Not Important: Did you have the latest main pulled in and did you run `yarn`? Just because this indentation has been an issue [I tried to fix](https://github.com/wellcomecollection/wellcomecollection.org/pull/11091) and I wonder if it's not fixed. 🤔